web/components/nav/FcAppbar.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <v-app-bar
+    app
+    class="fc-appbar"
+    color="white"
+    clipped-left
+    dense
+    role="banner">
```

Comment:
I don't _think_ we need the `role` here - it's at top-level and isn't inside other landmarks / sections.